src/plus/webviews/graph/graphWebview.ts

```diff
@@ -1087,6 +1119,53 @@ export class GraphWebview extends WebviewBase<State> {
 		return this.container.storage.getWorkspace('graph:columns');
 	}
 
+	private getHiddenRefs(): Record<string, GraphHiddenRef> | undefined {
+		return this.container.storage.getWorkspace('graph:hiddenRefs');
+	}
+
+	private async getHiddenRefsById(
+		hiddenRefs: Record<string, GraphHiddenRef> | undefined
+	): Promise<GraphHiddenRefs | undefined> {
+		if (hiddenRefs == null) return undefined;
+
+		// This validation has too much performance impact. So we decided to comment those lines
+		// for v13 and have it as tech debt to solve after we launch.
+		// See: https://github.com/gitkraken/vscode-gitlens/pull/2211#discussion_r990117432
+		// if (this.repository == null) {
+		// 	this.repository = this.container.git.getBestRepositoryOrFirst();
+		// 	if (this.repository == null) return undefined;
+		// }
+
+		// const [hiddenBranches, hiddenTags] = await Promise.all([
+		// 	this.repository.getBranches({
+		// 		filter: b => !b.current && hiddenRefs[b.id] != undefined,
+		// 	}),
+		// 	this.repository.getTags({
+		// 		filter: t => hiddenRefs[t.id] != undefined,
+		// 	}),
+		// ]);
+
+		// const filteredHiddenRefsById: GraphHiddenRefs = {};
+
+		// for (const hiddenBranch of hiddenBranches.values) {
+		// 	filteredHiddenRefsById[hiddenBranch.id] = hiddenRefs[hiddenBranch.id];
+		// }
+
+		// for (const hiddenTag of hiddenTags.values) {
+		// 	filteredHiddenRefsById[hiddenTag.id] = hiddenRefs[hiddenTag.id];
+		// }
+
+		// return filteredHiddenRefsById;
+
+		// For v13, we return directly the hidden refs without validating them
+		const hiddenRefsById: GraphHiddenRefs = {};
+		for (const [id, graphHiddenRef] of Object.entries(hiddenRefs)) {
+			hiddenRefsById[id] = {
+				...graphHiddenRef,
+			};
+		}
```

Comment:
Looks like this recreates the same object being passed-in. Is this work needed?

Reply:
Changed in commit: `Removed logic that recreates the same object`